pkg/kics/service.go

```diff
@@ -219,3 +222,21 @@ func prepareScanDocumentValue(bodyType map[string]interface{}, kind model.FileKi
 		}
 	}
 }
+
+func updateMaskedSecrets(vulnerabilities *[]model.Vulnerability, maskedSecretsTracked []secrets.SecretTracker) {
+	for idx, _ := range *vulnerabilities {
+		for _, secretT := range maskedSecretsTracked {
+			if (*vulnerabilities)[idx].FileName == secretT.ResolvedFilePath {
+				vulnLines := (*(*vulnerabilities)[idx].VulnLines)
```

Comment:
We should be careful with the `vulnLines` length. The default is 3 but could be more